src/pgduckdb_table_am.cpp

```diff
@@ -82,13 +82,12 @@ duckdb_scan_end(TableScanDesc sscan) {
 }
 
 static void
-duckdb_scan_rescan(TableScanDesc sscan, ScanKey key, bool set_params, bool allow_strat, bool allow_sync,
-                   bool allow_pagemode) {
+duckdb_scan_rescan(TableScanDesc, ScanKey, bool, bool, bool, bool) {
```

Comment:
Pretty much all of the arguments in this file should be commented out, not fully removed. Maybe even ignore the warning for this specific file.